app/Controllers/Http/ProfilesController.ts

```diff
@@ -0,0 +1,81 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Database from '@ioc:Adonis/Lucid/Database'
+import UserProfile from 'App/Models/UserProfile'
+import UpdateProfileValidator from 'App/validators/UpdateProfileValidator'
+import Utils from 'App/utils/utils'
+import { Gender } from 'Contracts/Enums'
+import {DateTime} from 'luxon'
+
+export default class ProfilesController {
+  public async createProfile({}: HttpContextContract) {
+    return 200
+  }
```

Comment:
Remove this and createprofile endpoint from here and the same from routes as well. we not doing any action

Reply:
done